src/Files.App/Actions/Open/OpenSettingsFileAction.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Files Community
+// Licensed under the MIT License.
+
+using Windows.Storage;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal sealed partial class OpenSettingsFileAction :  IAction
+	{
+		public string Label
+			=> Strings.EditSettingsFile.GetLocalizedResource();
+
+		public string Description
+			=> Strings.EditSettingsFileDescription.GetLocalizedResource();
+
+		public HotKey HotKey
+			=> new(Keys.OemComma, KeyModifiers.CtrlShift);
+
+		public RichGlyph Glyph
+			=> new("\uE8DA");
+
+		public async Task ExecuteAsync(object? parameter = null)
+		{
+			await SafetyExtensions.IgnoreExceptions(async () =>
```

Comment:
Probably it's better to show a simple ContentDialog when failed, what do you think?

Reply:
That won't work in the settings dialog.

Reply:
Like open when it can.

```c#
if (!VisualTreeHelper.GetOpenPopupsForXamlRoot(MainWindow.Instance.Content.XamlRoot).Any())
{
    var dialog = new ContentDialog() { ... };
    await dialog.ShowAsync();
}
```

Reply:
That can be done, I pushed a commit with this change.